generation/maisi/maisi_diff_unet_training_tutorial.ipynb

```diff
@@ -57,7 +57,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from IPython.display import Image, display
</pre><p>Could you please also move this line to the setup cell, thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1860/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Thank you. I updated this part in the recent PR.